src/bin/index.js

```diff
@@ -1,20 +1,24 @@
 #!/usr/bin/env node
 
 const { error } = require('../lib/log');
-require('../cli/check-node-version');
+const checkNodeVersion = require('../cli/check-node-version');
 
-const main = require('../lib/main');
+try {
+  checkNodeVersion();
+} catch (e) {
+  error(e.message);
+  process.exitCode = 1;
+}
```

Comment:
I didn't need to convert `checkNodeVersion` to a function but it felt more correct.